### PR TITLE
Standardize help page

### PR DESCRIPTION
**Pull request by j-smola**
# Description
## What changed?
Related JIRA issue: GDR-1911

## Why was it changed?
To standardize help page for packages to be released - user expirences after using `help("{pkgname}")` and `help(pakckage = "{pkgname}")`

(because this change is only for documentation - version of the package has not been bumped and no tests are required)

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [ ] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 .rplatform.cookie
 .Rproj.user
+.Rhistory
 .DS_Store
 */.DS_Store
 .github_access_token.txt
+src/*.o
```
Comment (gladkia) [inline .gitignore:7]: 🙏

File: DESCRIPTION [hunk 2]
```diff
@@ -52,4 +52,5 @@ Encoding: UTF-8
 LazyLoad: yes
 NeedsCompilation: yes
 RoxygenNote: 7.2.3
+Roxygen: list(markdown = TRUE)
```
Comment (gladkia) [inline DESCRIPTION:55]: 💪

File: man/add_CellLine_annotation.Rd [hunk 3]
```diff
@@ -25,7 +25,7 @@ add_CellLine_annotation(
 \item{DB_cellid_header}{string with colnames with cell line identifier
 in the annotation file}
 
-\item{DB_cell_annotate}{character vector with mandatory colnames 
+\item{DB_cell_annotate}{character vector with mandatory colnames
```
[the remark this answers is not in the document]
Comment (j-smola) [reply man/add_CellLine_annotation.Rd:28]: @gladkia The source of these all changes is here
```
Roxygen: list(markdown = TRUE) 
```
Most of them are removing the trailing space, there are a few with `href -mailto`, change `...` into `\code{...}` and descriptive itemized list.

More details - see [here](https://roxygen2.r-lib.org/articles/markdown.html)

**Review by gladkia — APPROVED** [review]
GJ!

**Review by darsoo — APPROVED** [review]

**Review by bczech — APPROVED** [review]

**Review by marcin-kam — APPROVED** [review]

**Pull request merged**